assets/styles/main.css

```diff
@@ -0,0 +1,264 @@
+  background-color: #3F403C;
+  padding: 0 3%;
+  height: 11vh;
+}
+
+header a {
+  color: #C4D3E5;
+}
+
+header a:hover {
+  color:  #AB89A5;
+}
+
+header h2,
+header li,
+header nav {
+  display: inline-block;
+  line-height: 11vh;
+}
+
+header h2 {
+  margin: 0;
+  font-size: 2em;
+}
+
+header nav {
+  float: right;
+}
+
+header ul {
+  margin: 0;
+  padding: 0;
+  font-size: 1em;
+  font-weight: bold;
+
+}
+
+header li {
+  text-decoration: none;
+  margin: 0 5px;
+}
+
+/*************************
+          HERO
+*************************/
+
+main #hero {
+  background-image: url("../../assets/imgs/succulent-hero.jpg");
+  background-size: cover;
+  width:100%;
+  height: 83vh;
+  background-position: center;
+}
+
+#hero h1 {
+  margin: 0;
+  font-size: 6em;
+  padding-top: 3%;
+}
+
+#hero {
+  text-align: center;
+}
+
+/*************************
+         PROJECTS
+*************************/
+
+#projects {
+  text-align: center;
+  background-image: url("../../assets/imgs/succulent4.jpg");
+  background-size: cover;
+  width: 100%;
+  height: 83vh;
+}
+
+#projects p {
+  text-align: left;
+  font-weight: bolder;
+}
+
+#projects h1 {
+  margin: 0;
+  font-size: 3.5em;
+  padding-top: 5%;
+}
+
+#projects section a h2 {
+  font-size: 1.8em;
+}
+
+.project a {
+  color: black;
+}
+
+.project:hover {
+  background-color: rgba(171, 137, 165, 0.8);
+}
+
+.project {
+  border-radius: 4em;
+  display: inline-block;
+  width: 25%;
+  vertical-align: top;
+  padding: 0 2%;
+  margin: 5% 1%;
+  background-color: rgba(255, 248, 241, 0.8);
+  height: 45vh;
+}
+
+/*************************
+         ABOUT
+*************************/
+
+#about {
+  background-image: url("../../assets/imgs/cacti.jpg");
+  background-size: cover;
+  width:100%;
+  height: 83vh;
+  background-position: center;
+}
+
+#about section {
+  padding: 6% 3%;
+  text-align: center;
+}
+
+#about img {
+  border-radius: 50%;
+  padding: 0;
+  -webkit-filter: grayscale(100%); /* Safari 6.0 - 9.0 */
+  filter: grayscale(100%);
+}
+
+#about h1 {
+  margin: 0;
+}
+
+#about img, #about p {
+  display: inline-block;
+  vertical-align: middle;
+  padding: 0 3%;
+}
+
+#about p {
+  width: 70vh;
+  margin: 0;
+  border-left: 2px solid black;
+}
+
+/*************************
+          TRAVEL
+*************************/
+
+#travel section:first-child {
+  text-indent: 0;
+}
+#travel section ul li{
+  display: inline-block;
+}
+
+#travel section nav {
+  text-align: center;
+}
+
+#travel section nav{
+  height: 100vh;
+  background-image: url("../../assets/imgs/worldhands.jpg");
+  background-size: cover;
+}
+
+#travel section a {
+  color: black;
+}
+
+#travel section a:hover {
+  color: #7A4D7F;
+}
+
+#travel section ul {
+  margin: 0;
+  padding: 3% 0 0 0;
+  font-size: 2em;
+}
+
+#travel section li:nth-of-type(2)::before,
+#travel section li:last-of-type::before {
+  content: " | "
+}
+
+/*************************
+          FOOTER
+*************************/
+
+footer {
+  height: 5vh;
```

Comment:
How could you change the height setting so the footer does not shrink (and the text stop fitting inside of it) when the window is made smaller?